db/schema.rb

```diff
@@ -90,10 +90,10 @@
     t.string   "processing_state",        default: "requested", null: false
     t.datetime "created_at",                                    null: false
     t.datetime "updated_at",                                    null: false
-    t.string   "reference_no"
-    t.boolean  "closed"
     t.boolean  "override_delivery_error", default: false
     t.string   "delivery_error_type"
+    t.string   "reference_no"
+    t.boolean  "closed"
```

Comment:
Someone needs to write a better schema generator for ActiveRecord that doesn't muddle the order. If it didn't bother trying to line up the columns, the diffs would be less volatile, too. Perhaps I'll do that …